libraries/core-react/stories/components/Tooltip.stories.tsx

```diff
@@ -34,104 +21,283 @@ const TextWrapper = styled.div`
-        <Typography variant="body_long">
+        <Typography variant="body_long" style={{ marginTop: 8 }}>
           If you have Safari users, you will need to add inline style to your
-          disabled element, shown in the example below. This will help trigger
-          the mouse events correctly. Unfortunately, this workaround overwrites
-          the &apos;not-allowed&apos; cursor.
+          disabled element, shown in the example below, as well as wrapping the
+          anchor inside a div which will handle the pointer event handlers. This
+          will help trigger the mouse events correctly. Unfortunately, this
+          workaround overwrites the &apos;not-allowed&apos; cursor.
         </Typography>
       </TextWrapper>
       <Wrapper>
-        <Tooltip title="Tooltip" placement="topLeft">
-          <Button disabled>Disabled</Button>
-        </Tooltip>
+        <Button
+          disabled
+          aria-describedby="tooltip-disabled-chrome"
+          ref={referenceElementOne}
+          onPointerEnter={() => handleOpen(1)}
+          onPointerLeave={handleClose}
+        >
+          Disabled, but hover works!
+        </Button>
+
+        <Tooltip
+          id="tooltip-disabled-chrome"
+          open={openState === 1}
+          title="Tooltip works!"
+          anchorEl={referenceElementOne.current}
+        />
 
-        <Tooltip title="Tooltip" placement="topLeft">
-          <Button style={{ pointerEvents: 'none' }} disabled>
-            Disabled for Safari Browser
+        <div
+          ref={referenceElementTwo}
+          onPointerEnter={() => handleOpen(2)}
+          onPointerLeave={handleClose}
+          aria-describedby="tooltip-disabled-safari"
+        >
+          <Button disabled style={{ pointerEvents: 'none' }}>
+            Just Safari example
           </Button>
-        </Tooltip>
+        </div>
+
+        <Tooltip
+          id="tooltip-disabled-safari"
+          open={openState === 2}
+          title="Tooltip works!"
+          anchorEl={referenceElementTwo.current}
+        />
       </Wrapper>
-    </>
+    </Body>
+  )
+}
+
+type Data = {
+  number: string
+  description: string
+  origin: string
+  price: number
+}
+
+const data: Data[] = [
+  {
+    number: '123-456',
+    description: 'Pears',
+    origin: 'Europe',
+    price: 1.5,
+  },
+  {
+    number: '234-567',
+    description: 'Apples',
+    origin: 'Africa',
+    price: 1.2,
+  },
+  {
+    number: '45-6789',
+    description: 'Oranges',
+    origin: 'South America',
+    price: 1.8,
+  },
+  {
+    number: '67-890',
+    description: 'Kiwi',
+    origin: 'Australia',
+    price: 2.1,
+  },
+  {
+    number: '89-012',
+    description: 'Mango',
+    origin: 'South Africa',
+    price: 2.5,
+  },
+  {
+    number: '89-012',
+    description: 'Pineapple',
+    origin: 'Paraguay',
+    price: 1.9,
+  },
+  {
+    number: '89-012',
+    description: 'Pomegranate',
+    origin: 'Persia',
+    price: 4.5,
+  },
+]
```

Comment:
Perhaps this table data could be shared between several stories in a data folder?